src/Core/Framework/RateLimiter/RateLimiterFactory.php

```diff
@@ -72,9 +72,8 @@ public function create(?string $key = null): LimiterInterface
         // prevent symfony errors due to customized values
         /** @var RateLimiterConfig $rateLimiterConfig */
         $rateLimiterConfig = \array_filter($this->config, static fn ($key): bool => !\in_array($key, ['enabled', 'reset', 'cache_pool', 'lock_factory', 'limits'], true), \ARRAY_FILTER_USE_KEY);
-        $this->config = $rateLimiterConfig;
 
-        $sfFactory = new SymfonyRateLimiterFactory($this->config, $this->storage, $this->lockFactory);
+        $sfFactory = new SymfonyRateLimiterFactory($rateLimiterConfig, $this->storage, $this->lockFactory);
```

Comment:
Hey @JoshuaBehrens 

thanks for your contribution! :blue_heart: 

Can you add a unit test for your change?

Reply:
Sure, I just wanted to have it already posted :) will add changelog and test

Reply:
I likely want to rebase as well :)